index.Rmd

```diff
@@ -58,7 +58,7 @@ knitr::write_bib(c(
 ```
 
 
-```{r html, echo=FALSE}
```

Comment:
@WillemSleegers  huh, this never should have been labeled 'html' in the first place.

 Fwiw in some context I used html *blocks* to enable a sort of commenting ... as the visual editor in Rstudio messes up the `<!-- Hi I am a comment -->` stuff